django_project/ford3/templates/provider_form.html

```diff
@@ -3,7 +3,7 @@
 
 {% block body %}
 	<div class="container">
-		<form action="/providers/{{ provider_id }}/edit" method="post">
+		<form action="/providers/{{ provider_id }}/edit" method="post" enctype="multipart/form-data">
```

Comment:
Just leave the action empty to refer to current url :).
If you insists on generating the exact urls, use `reverse`